TAF/taf/IORResolver_T.cpp

```diff
@@ -45,7 +45,7 @@ namespace TAF
 
     template <typename T>
     typename T::_var_type
-    IORResolver_T<T>::getResult(void)
+    IORResolver_T<T>::getResult(void) const
     {
         return T::_duplicate(this->result_.in());
```

Comment:
This could just as easily be return this->result_; as a var to var copy will do the duplicate and *may* be more efficient under Return Value Optimisation (RVO) rules.